source/unified-test-format/unified-test-format.md

```diff
@@ -1133,6 +1136,27 @@ The structure of this object is as follows:
 - `connectionCheckedInEvent`: Optional object. If present, this object MUST be an empty document as all fields in this
     event are non-deterministic.
 
+<span id="expectedEvent_connectionPendingResponseStarted"></span>
+
+- `connectionPendingResponseStarted`: Optional object. If present, this object MUST be an empty document as all fields
+    in this event are non-deterministic.
+
```

Comment:
The values are non-deterministic, but we can still assert on the structure of the new CMAP events. Right?

It might be worth instructing drivers to assert that properties exist on the events and that they're of the expected type but keep the phrasing that prohibits drivers from asserting on the values.

Reply:
We simply follow other check out events here, like `connectionCheckedOutEvent`. I can add simple validation for properties to exist.